girder_monkeybrains/web_client/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "@girder/monkeybrains",
+    "version": "1.0.0",
+    "peerDepencencies": {
+        "@girder/core": "*"
```

Comment:
This should use version `^3`. There are v2 releases in that repo (and could possibly be more in the future).

Reply:
I'm not sure what you are saying.

    "@girder/core": "^3"

?

This frontend stuff is pure cargo cult for me.

Reply:
IMO trying to specify versions for peer dependencies causes more harm then good even when they are applied correctly.  For this reason, I didn't bother with it in the core plugins or the migration docs.